ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -40,21 +49,15 @@ const renderFirstNotification = (notification, idRefMap) => {
       key={`whats-new-popop-notificatiion-${id}`}
       ref={idRefMap[id]}
     >
-      {image && (
-        <img
-          className="whats-new-popup__notification-image"
-          src={image.src}
-          height={image.height}
-          width={image.width}
-        />
-      )}
+      {!placeImageBelowDescription && imageComponent ? imageComponent : null}
```

Comment:
Could this be refactored to `{!placeImageBelowDescription && imageComponent}`?

Reply:
yes, thanks, refactored and pushed 7c8fe41da